libs/infinity_emb/Dockerfile

```diff
@@ -39,75 +47,14 @@ RUN poetry install --no-interaction --no-ansi --extras "${EXTRAS}"  --without li
 # remove cache
 RUN poetry cache clear pypi --all
 
-FROM builder as testing
-# install lint and test dependencies
-RUN poetry install --no-interaction --no-ansi --extras "${EXTRAS}"
-# lint 
-RUN poetry run ruff .
-RUN poetry run black --check .
```

Comment:
Since you have removed the black and ruff check, can you add it as a github action to be run on PRs?

Reply:
Made the change directly to Infinity and pulled the changes. Has ruff check in as part of CI